backend/controllers/election.js

```diff
@@ -0,0 +1,16 @@
+import { createApiResponse } from "../helpers/createApiResponse.js";
+import { electionFactoryService } from "../services/electionService.js";
+
+const createElection = async (req, res, next) => {
```

Comment:
this controller is not supposed to return all of the ballots. However, it is reasonable to return the created ballot, though that requires a change in blockchain code, and will probably be done in a different pr. For now clear up the controller and return the data that is passed in from the body